site-src/geps/gep-713.md

```diff
@@ -0,0 +1,496 @@
+  policies that had been directly applied to the resource. This could get
+  confusing as it would not provide any insight into policies attached above or
+  below.
+* Since we only control some of the resources a policy might be attached to,
+  adding policies to status would only be possible on Gateway API resources, not
+  Services or other kinds of backends.
+
+Although these concerns are not unsolvable, they lead to the conclusion that
+a Kubectl plugin should be our primary approach to providing visibility here,
+with a possibility of adding policies to status at a later point.
+
+### Interaction with Custom Filters and other extension points
+There are multiple methods of custom extension in the Gateway API. Policy
+attachment and custom Route filters are two of these. Policy attachment is
+designed to provide arbitrary configuration fields that decorate Gateway API
+resources. Route filters provide custom request/response filters embedded inside
+Route resources. Both are extension methods for fields that cannot easily be
+standardized as core or extended fields of the Gateway API. The following
+guidance should be considered when introducing a custom field into any Gateway
+controller implementation:
+
+1. For any given field that a Gateway controller implementation needs, the
+   possibility of using core or extended should always be considered before
+   using custom policy resources. This is encouraged to promote standardization
+   and, over time, to absorb capabilities into the API as first class fields,
+   which offer a more streamlined UX than custom policy attachment.
+
+2. Although it's possible that arbitrary fields could be supported by custom
+   policy, custom route filters, and core/extended fields concurrently, it is
+   strongly recommended that implementations not use multiple mechanisms for
+   representing the same fields. A given field should only be supported through
+   a single extension method. An example of potential conflict is policy
+   precedence and structured hierarchy, which only applies to custom policies.
+   Allowing a field to exist in custom policies and also other areas of the API,
+   which are not part of the structured hierarchy, breaks the precedence model.
+   Note that this guidance may change in the future as we gain a better
+   understanding for extension mechanisms of the Gateway API can interoperate.
+
+### Advantages
+* Incredibly flexible approach that should work well for both ingress and mesh
+* Conceptually similar to existing ServicePolicy proposal and BackendPolicy
+  pattern
+* Easy to attach policy to resources we don’t control (Service, ServiceImport,
+  etc)
+* Minimal API changes required (just expansion of status)
+* Simplifies packaging an application for deployment as policy references do not
+  need to be part of the templating
+
+### Disadvantages
+* May be difficult to understand which policies apply to a request
+
+## Removing BackendPolicy
```

Comment:
Conceivably, BackendPolicy could end up as the example Policy object, provided under the project's api group?

Probably outside the scope of this design though.

Reply:
Yeah I agree with that. I think we could/should try to find enough portable concepts that could fit in a shared policy resource, but don't think that BackendPolicy provides any value as it exists today. I think it was mostly meant to establish a pattern for policy attachment, but the actual config included is so limited that I don't think it would be broadly used.